### DISCUSSION TITLE
Automatic interactive API documentation not working as expected.

### DISCUSSION THREAD

**Discussion by Lepiloff** [fastapi/fastapi#7369] [Questions] [answered]
I've come across some strange behavior in automatic interactive API documentation (provided by Swagger UI)
when I get to the page
`http://127.0.0.1:8000/docs`
The list of routes is displayed with the parameters I have set: `prefix`, `tags`. So far, so good. 

![Screenshot from 2020-07-22 18-43-12](https://user-images.githubusercontent.com/28572064/88197701-825d7d80-cc4b-11ea-9607-79a50fa21e7c.png)

Problems begin when I click on any of the methods. A new group with the` tags=default` parameter appears on the page and the endpoint that I clicked is automatically added to it. By successively clicking on all my endpoints I end up with one group with tags=default in which all my endpoints are concentrated. But I can't see their contents, because when I click on any of them I see an infinitely spinning boot. 

![Screenshot from 2020-07-22 18-46-07](https://user-images.githubusercontent.com/28572064/88197880-c0f33800-cc4b-11ea-9a09-ca6de698344b.png)
This behavior is observed on both the local machine and the remote server.
In the **main.py** file, I use this routing system
```
app.include_router(text_ocr.router, prefix='/image', tags=['image'])
app.include_router(notes.router, prefix='/notes', tags=['notes'])
app.include_router(product.router, prefix='/product', tags=['product'])
```
For the purity of the experiment, I commented on these call points and added a standard endpoint from the documentation
```
# app.include_router(text_ocr.router, prefix='/image', tags=['image'])
# app.include_router(notes.router, prefix='/notes', tags=['notes'])
# app.include_router(product.router, prefix='/product', tags=['product'])
@app.get("/")
async def root():
    return {"message": "Hello World"}
```
But I got the same result.

**Comment by tanzerm** [comment]
I have the same problem but only in Chrome. In Firefox everything works as expected.

**Comment by whwright** [comment]
I am seeing this issue in firefox/chrome/safari on instances that were deployed days ago. The "authorize" button in the top right has also disappeared.

**Comment by Lepiloff** [comment]
@tanzerm Same problem in Firefox

**Comment by Kludex** [comment, marked as answer]
This issue was solved in **FastAPI 0.60.1**. 

Please upgrade your FastAPI version.

**Comment by Lepiloff** [comment]
Confirmed, after the version update, the problem has disappeared.

**Comment by tanzerm** [comment]
@Kludex Thank you, this solved the problem for me!

**Comment by rjcabrera158** [comment]
I am still experiencing the same problem even with the latest version with FastAPI 0.60.1

**Comment by Kludex** [comment]
@rbcabreraUPM It may be useful to say that you were also facing this issue: https://github.com/tiangolo/fastapi/issues/1780

As we see by the comments above, others are not facing the swagger issue in **FastAPI 0.60.1**. I may suggest you to check your environment setup.

**Comment by rjcabrera158** [comment]
> @rbcabreraUPM It may be useful to say that you were also facing this issue: #1780
> 
> As we see by the comments above, others are not facing the swagger issue in **FastAPI 0.60.1**. I may suggest you to check your environment setup.

I got it working already. dont know why it wasn't working before. thanks for the help!

**Comment by tiangolo** [comment]
Thanks for the help here everyone! :clap: :bow: 

Thanks for reporting back and closing the issue :+1:
